TimberAPI/PluginSystem/PluginLocationService.cs

```diff
@@ -0,0 +1,95 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using BepInEx.Bootstrap;
+
+namespace TimberbornAPI.PluginSystem
+{
+    public class PluginLocationService
```

Comment:
A serious lack of documentation of the resposibilities of the class and methods here